src/sdk/pynni/nni/compression/torch/utils.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+def validate_op_names(model, op_names, logger):
+    found_names = set()
+    for name, _ in model.named_modules():
+        found_names.add(name)
+
+    not_found_op_names = list(set(op_names) - found_names)
```

Comment:
inconsistent: there is `list` here, but no in line 20.

Reply:
Thanks,  list added for better print.